travel_platform/travel_app/static/travel_app/js/common.js

```diff
@@ -0,0 +1,8 @@
+$(function () {
```

Comment:
Well, I suggest to create something like:

```
function my_func(element_id) {
  $(element_id).submit(function(e) {
    e.preventDefault();
    ...
  }
}

```

And then use this function for different elements (`my_func("#search-city-form"))`, `my_func("#museums-city")`). I guess, it should be usefull.